src/app/CMakeLists.txt

```diff
@@ -104,7 +104,9 @@ endif()
 ###########################################
 # Resources
 ###########################################
-qt_add_resources(APP_RCC_SOURCES app.qrc)
+qt_add_resources(APP_RCC_SOURCES
+    app.qrc
+    ${PROJECT_SOURCE_DIR}/images/resources.qrc)
```

Comment:
hmm, can we move it to `projectscene` module?

Reply:
fixed 👍